### ISSUE TITLE
Loading Pre-Trained LightGBM model doesn't correctly load Label metadata for key <-> value.

### ISSUE THREAD

**Issue by michaelgsharp** [dotnet/machinelearning#6686]
Loading Pre-Trained LightGBM model doesn't correctly load Label metadata for key <-> value. Need to parse the labels out of the model file and add a new annotation for it.
